inc/sys/wait.h

```diff
@@ -1 +1,16 @@
-#pragma once
+#ifndef __WAIT_H__
```

Comment:
Maybe we can use here  _SYS_WAIT_H like in original one. It is more unique.

Reply:
I'd go with __SYS_WAIT_H__